media-libs/freeimage/files/freeimage-3.18.0-null-to-false.patch

```diff
@@ -0,0 +1,18 @@
+841973
```

Comment:
https://bugs.gentoo.org/841973 instead please

Also, I'd link to the Alpine patch, because why not? (Could say "See also: ...")